src/views/Actors/components/ActorTable/ActorTable.tsx

```diff
@@ -36,9 +36,9 @@ const TableWrapper = styled.div({
 const ContainerList = styled.div({
```

Comment:
We should migrate to the `@material/styles` instead of continue using @emotion/styles

Reply:
done